src/lib/context-menu.ts

```diff
@@ -256,6 +257,32 @@ export class ContextMenu {
       )
     }
 
+    if (type === schema.nodes.table_element) {
+      // Check if the selection is inside the table.
+      const isInTable = hasParentNodeOfType(schema.nodes.table)(
+        this.view.state.selection
+      )
+
+      if (isInTable) {
```

Comment:
@mbartenev-atypon, the other option is to have the `Add Reference Note`  menu item disabled. But I think we usually don't have disabled items in context-menu and we just add the options(actions) that we can do. Am I right?

Reply:
Yes, I think so .

Reply:
Got it, thanks